File: QTD.SPA/src/app/_DtoModels/EmployeePosition/EmployeePositionUpdateOptions.ts
export class EmployeePositionUpdateOptions {
  employeePositionId?:any;
  employeeId!: any;
  positionId!: any;
  startDate!: Date | string;
  endDate!: Date | string | null;
  trainee!: boolean;
  qualificationDate!: Date | string | null ;
  managerName?:string;
  trainingProgramVersion?:string;
  isCertificationRequired?:boolean;
  isSignificant?:boolean;
}

File: QTD2.Infrastructure/Model/EmployeePosition/EmployeePositionUpdateOptions.cs
using System;

namespace QTD2.Infrastructure.Model.EmployeePosition
{
    public class EmployeePositionUpdateOptions
    {
        public int EmployeeId { get; set; }

        public int PositionId { get; set; }

        public DateOnly StartDate { get; set; }

        public DateOnly? EndDate { get; set; }

        public bool Trainee { get; set; }

        public DateOnly? QualificationDate { get; set; }

        public string ManagerName { get; set; }

        public string TrainingProgramVersion { get; set; }

        public bool? IsCertificationRequired { get; set; }
        public bool IsSignificant { get; set; }
        public int EmployeePositionId { get; set; }
    }
}
